external/fv3fit/tests/training/test_graph.py

```diff
@@ -0,0 +1,117 @@
+import dataclasses
```

Comment:
Given you followed the same testing pattern as in `test_train.py`, would it make sense to add this test to that file, instead of making a new file? It would de-duplicate the code and so be easier to maintain, if you're able to.